### PR TITLE
Update devops-command-center.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Formatting and punctuation changes, revised instructions for finding ! icon in header.

### CODE REVIEW

File: source/administration/devops-command-center.rst [hunk 1]
```diff
@@ -133,15 +133,20 @@ You can view all incidents in the incident Backstage via **Main Menu > Incidents
 - A link icon to open the corresponding incident channel
 - Status (**Ongoing** or **Ended**)
 - The commander, including profile picture and username
-- A prompt to `export the incident channel <#exporting-channels>`_
+- A prompt to `**Export the Incident Channel** <#exporting-channels>`_
```
Comment (amyblais) [inline source/administration/devops-command-center.rst:136]: Bolding formatting doesn't work here

File: source/administration/devops-command-center.rst [hunk 2]
```diff
@@ -133,15 +133,20 @@ You can view all incidents in the incident Backstage via **Main Menu > Incidents
...
 - The total duration
+  
+  - The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it shows the end time (in the user's timezone).
+
 - The number of members currently involved in the incident
+
+  - The **Members Involved** widget indicates the total number of users that participated in the channel, either
+by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
```
Comment (amyblais) [inline source/administration/devops-command-center.rst:144]: This doesn't need to be in a separate line

File: source/administration/devops-command-center.rst (hunk 1 above)
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:136]:
```suggestion
- A prompt to `Export the Incident Channel <#exporting-channels>`_
```

Comment (justinegeffen) [reply source/administration/devops-command-center.rst]: Sorted, thanks!

File: source/administration/devops-command-center.rst (hunk 2 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:145]: The formatting was a bit off, so I reformatted and it looks better.

Comment (amyblais) [reply source/administration/devops-command-center.rst:145]: In the bullet point for ``- The **Members Involved** widget indicates the total number of users that participated in the channel, either by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.``, the paragraph is broken into a separate line at ``by posting a message...`` so the formatting is still a little off.

Comment (justinegeffen) [reply source/administration/devops-command-center.rst:145]: That's really annoying. I'll pull this locally and fix it, thanks @amyblais.

Comment (amyblais) [reply source/administration/devops-command-center.rst]: Great!

File: source/administration/devops-command-center.rst [hunk 3]
```diff
@@ -24,29 +24,29 @@ Starting incidents
...
 The commander of the incident can:
 
-- Select a playbook when starting an incident
-- Create checklists
-- Invite people to the channel
-- Hand over the commander role to another channel member
+- Select a playbook when starting an incident.
+- Create checklists.
+- Invite people to the channel.
+- Hand over the commander role to another channel member.
```
Comment (agarciamontoro) [inline source/administration/devops-command-center.rst:42]: I *never* know what the punctuation in lists should be. We can discuss in the Language channel in Mattermost! :)

File: source/administration/devops-command-center.rst [hunk 4]
```diff
@@ -120,29 +120,33 @@ To view details of active incidents, select **! Incidents** from the plugin menu
...
 If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the commander name and incident checklist are displayed. If you're a participant in an incident channel, you can modify incident details from within that channel.
 
 You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks** and then select an incident to view the following details:
 
 - Incident name
+
 - A link icon to open the corresponding incident channel
+
 - Status (**Ongoing** or **Ended**)
+
```
Comment (agarciamontoro) [inline source/administration/devops-command-center.rst:137]: Just out of curiosity: why are adding blank lines here?

File: source/administration/devops-command-center.rst [hunk 5]
```diff
@@ -120,29 +120,33 @@ To view details of active incidents, select **! Incidents** from the plugin menu
...
+- A prompt to `Export the Incident Channel <#exporting-channels>`_
+
+- The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it shows the end time (in the user's timezone).
+
+- The **Members Involved** widget indicates the total number of users that participated in the channel, either by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
+
 - The total number of messages displayed includes messages posted by both users and bots (including the incident response bot). It does not include system or ephemeral messages.
 
+- A graph depicting when each checklist item was completed
```
Comment (agarciamontoro) [inline source/administration/devops-command-center.rst:148]: Typo?

```suggestion
- A graph depicting when each checklist item was completed.
```

File: source/administration/devops-command-center.rst (hunk 3 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:42]: Great catch. In cases like this where there are shorter and longer sentences, I use periods for each line for consistency. For lists with single words/short phrases I don't use punctuation. I have noted that this is missing from our style guide so I'll add that in, thanks. 

As an aside, [this article](https://intelligentediting.com/blog/how-to-punctuate-bulleted-and-numbered-lists/#:~:text=There%20is%20no%20single%20right,take%20different%20forms%20of%20punctuation.&text=a%20semicolon%20otherwise) has some guidance.

File: source/administration/devops-command-center.rst (hunk 4 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst]: Formatting error; fixed now! :)

File: source/administration/devops-command-center.rst (hunk 3 above)
Comment (agarciamontoro) [reply source/administration/devops-command-center.rst:42]: Oh gosh, thank you for that link!

**Pull request merged**